app/sidekiq/event_bus_gateway/va_notify_email_status_callback.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module EventBusGateway
+  class VANotifyEmailStatusCallback
+    def self.call(notification)
+      status = notification.status
+
+      add_metrics(status)
+      add_log(notification) if notification.status != 'delivered'
+    end
+
+    def self.add_log(notification)
+      context = {
+        notification_id: notification.notification_id,
+        source_location: notification.source_location,
+        status: notification.status,
+        status_reason: notification.status_reason,
+        notification_type: notification.notification_type
+      }
+
+      Rails.logger.error(name, context)
+    end
+
+    def self.add_metrics(status)
+      case status
+      when 'delivered'
+        StatsD.increment('api.vanotify.notifications.delivered')
+        StatsD.increment('callbacks.event_bus_gateway.va_notify.notifications.delivered')
+        tags = ['service:event-bus-gateway', 'function: Event Bus Gateway - VA Notify decision letter ready email']
```

Comment:
Should this have `'service:eventbus-gateway'` (no space between event and bus)? Same applies to L34.